docs/db-import.md

```diff
@@ -1,21 +1,21 @@
 # Automatic database import
 
-The `db` service container can perform automatic import of database dump upon initialization.
+The `db` service container can perform an automatic import of the database dump upon initialization.
 
 # Setting up
 
 #### 1. Create a folder for database dumps
 
-Create a folder for database dumps inside project root called `db` (name can differ).
+Create a folder for database dumps inside the project root called `db` (name can differ).
 
-#### 2. Put `*.sql` or `*.sql.gz` file(s) into the newly created `db` folder.
+#### 2. Add `*.sql` or `*.sql.gz` file(s) into the newly created `db` folder.
 
-!!! note "You can put multiple files" 
-    You can put multiple `*.sql` and `*.sql.gz` files. They will all be imported in an alphabetical order. Database that is set in `MYSQL_DATABASE` variable is used.
+!!! **Note:**
```

Comment:
`!!! note` and similar constructions should not be altered. This is a special syntax for readthedocs. See http://docksal.readthedocs.io/en/develop/db-import/